Gemfile

```diff
@@ -1,9 +1,10 @@
 source "http://rubygems.org"
+ruby "1.9.3"
```

Comment:
ruby 1.9?

Reply:
Yeah this is the version of ruby that this library was built with. Modern versions of ruby got me lost in dependency hell, and older versions of ruby segfaulted when installing the gems. I had to experiment and try quite a few different versions of ruby in order to find the one they apparently built this gem with. :laughing:

Reply:
wow, shows how old this gem is

Reply:
for sure, also helps explain how so many zip codes were missing from their database